lib/index.js

```diff
@@ -68,19 +68,40 @@ internals.loadExtras = function (args) {
     }
 };
 
+internals.jsParser = function (manifestPath) {
+    try {
```

Comment:
I brought this up earlier.  It looks like travis did fail but for some reason returned a 0 exit code.  @geek is investigating.